metrics-collector/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Env Vars:
+# REGISTRY: name of the image registry/namespace to store the images
+# NOCACHE: set this to "--no-cache" to turn off the Docker build cache
+#
+# NOTE: to run this you MUST set the REGISTRY environment variable to
+# your own image registry/namespace otherwise the `docker push` commands
+# will fail due to an auth failure. Which means, you also need to be logged
+# into that registry before you run it.
+
+set -ex
```

Comment:
My advice is to use a safer definition, I do `set -euo pipefail`, this setting is well documented on the internet and also I've seen others using it.